README.md

```diff
@@ -80,7 +81,7 @@ configuration to the file `./config.yaml` (i.e. in the current directory). This
 version > 2.0.0 and <= 2.1.2 you can use the older `run_topostats --create-config` option.
 
 ```bash
-run_topostats --create-config-file config.yaml
+run_topostats --create-config -f config.yaml
```

Comment:
Good spot! This is actually completely redundant now as we have removed `run_topostats` as of Pull Request #985.

I've created a small issue which I will address tomorrow to fix this (as its late and I'm not going to do it now).